zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -89,43 +80,41 @@
               </span>
             </div>
           </form>
+        <li class="nav-component">
+          <div class="dropdown">
+            <button class="btn dropdown-toggle" type="button" data-toggle="dropdown" style="margin-right: 5px;"
```

Comment:
@minahlee You removed the truncated text, but didn't implement it in css.
Could you add a class with this inside to this element?
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
max-width: 180px;
```